firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -129,13 +129,18 @@ void home()
 // A specific motor is passed to this function since we are only moving one here
 // Returns a 'MovementStatus' code, '0' if successful, varying nonzero values for various error codes
 // For all status codes, check 'MovementStatus' in chessboard.ino
-uint8_t moveStraight(uint8_t motor[], int startCol, int startRow, int endCol, int endRow)
+uint8_t moveStraight(uint8_t motor[], int endCol, int endRow)
 {
   // How many steps per space
   int dir, numSteps, unitSpaces;
   int i;
   int eighthStepsPerPulse;
   int *currentMotorPos;
+  float startCol, startRow;
+
+  // Converts current position to be in terms of unit spaces instead of eighth steps
+  startCol = currentX / (stepsPerUnitSpace * 8);
+  startRow = currentY / (stepsPerUnitSpace * 8);
```

Comment:
Here and below, it would be good to have a check to make sure that `currentX` and `currentY` are aligned with a unit space so that we know if something funky is going on with the positioning of the electromagnet. Right now, if they are not aligned, we have no way of being able to tell. @LSaber36 said that he prefers to address it later, here is a link to an issue regarding that change: #147

Reply:
After considering it, I agree with Nashir. Though, it would be a good idea to make this in a different issue, since it will involve either adding an error code, which means enums, or error correction, which involves some movement code.